src/Main.purs

```diff
@@ -83,20 +83,20 @@ main = HA.runHalogenAff do
   -- Finally, we'll attempt to fill the reference with the user profile associated with the token in
   -- local storage (if there is one). We'll read the token, request the user's profile if we can, and
   -- if we get a valid result, we'll write it to our mutable reference.
-  -- 
-  -- Note: this is quite a verbose request because it uses none of our helper functions. They're 
+  --
+  -- Note: this is quite a verbose request because it uses none of our helper functions. They're
   -- designed to be run in `AppM` (we're in `Effect`). This is the only request run outside `AppM`,
   -- so it's OK to be a little verbose.
   liftEffect readToken >>= traverse_ \token -> do
     let requestOptions = { endpoint: User, method: Get }
     res <- liftAff $ request $ defaultRequest baseUrl (Just token) requestOptions
-    let u = decodeAt "user" =<< lmap printResponseFormatError res.body
+    let u = decodeAt "user" =<< bimap printError _.body res
```

Comment:
This line and the `Data.Bifunctor` import are the only actual changes in this file. The rest is noise from my editor removing trailing spaces.

Reply:
Also, I was thinking this could be simplified, since we throw away any `Left` on the next line, anyway. I just wrote it this way to preserve the functionality of the original.

Reply:
Another option, since we throw away any `Left` on the next line, anyway:
```purescript
    let u = decodeAt "user" =<< rmap _.body res
```